.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
remove the issue template folder and all files in it

Reply:
done